pythoncz/templates/beginners_cs.html

```diff
@@ -116,8 +114,7 @@ <h3>
             <li><a href="https://godjango.com/">Screencasty pro webový framework Django</a> {{ en }}</li>
             <li><a href="https://docs.djangoproject.com/">Dokumentace k webovému frameworku Django</a> {{ en }}</li>
             <li><a href="http://interactivepython.org/runestone/static/pythonds/index.html">Základní algoritmy - v Pythonu a interaktivně</a></li>
-            <li><a href="http://inventwithpython.com/blog/2012/07/09/16-common-python-runtime-errors/">Vysvětlení nejčastějších chyb začátečníků</a> {{ en }}</li>
-            <li><a href="http://jessenoller.com/good-to-great-python-reads/">Odkazy na vysvětlení zajímavých či pokročilých detailů jazyka</a> {{ en }}</li>
```

Comment:
Je k něčemu https://web.archive.org/web/20170912115918/http://jessenoller.com/good-to-great-python-reads/ ?